src/simulate.jl

```diff
@@ -8,6 +8,28 @@ struct NestedFilterSimulation{T1, T2, T3, T4, T5, T6, T7}
     epsps::T7
 end
 
+struct Map{T1, T2, T3, T4, T5}
```

Comment:
You won't need `Map`, `Entropies`, and `Results` anymore after implementing `Recording` (see other comment).

Reply:
Structures removed (as well as function save_results)